### DISCUSSION TITLE
Problem with the process of installing oh-my-posh in Powershell 7.2.2

### DISCUSSION THREAD

**Discussion by ghost** [JanDeDobbeleer/oh-my-posh#1948] [Troubleshoot]
Hello, I have faced a weird problem, I have surfed the internet for hours searching for a solution and I couldn't find one.
I'm using Powershell 7.2.2

https://i.imgur.com/grlc0Jk.png

**Comment by JanDeDobbeleer** [comment]
That makes no sense unless the Gallery is down. I suggest installing via winget or scoop. Same result in the end.

**Reply by ghost** [reply]
do you know what is really baffling? is that this problem only appear with Powershell 7.2.2 but not with Windows Powershell 5.1

https://i.imgur.com/fTcGzA7.png

**Comment by mattcargile** [comment]
What happens when you run `Find-Module oh-my-posh` in Powershell 7.2.2?

**Reply by ghost** [reply]
this 

https://i.imgur.com/cSCIROt.png

sorry for not replying fast...

**Reply by JanDeDobbeleer** [reply]
@kiemkist same issue: https://stackoverflow.com/questions/68034974/powershell-scope-parameter-not-found

**Reply by ghost** [reply]
Oooooo i will look into that tomorrow, good night😴😴

Thanks🌹

**Reply by mattcargile** [reply]
Isn't that a `powershell.exe` 5.1 problem in the SO post?

**Reply by mattcargile** [reply]
Maybe there is something going on with your `$env:PSModulePath`? What does `Get-InstalledModule oh-my-posh -AllVersions` show?
